interface-definitions/interfaces_openvpn.xml.in

```diff
@@ -14,6 +14,133 @@
             <format>vtunN</format>
             <description>OpenVPN interface name</description>
           </valueHelp>
+          <childSpecification>
+            <requiredChildren>
+              <child>mode</child>
+            </requiredChildren>
+            <!-- OpenVPN client mode -->
+            <oneWayDependantChildren>
+              <dependants><descendant name="mode"><value>client</value></descendant></dependants>
+              <dependees>
+                <child>remote_host</child>
+              </dependees>
+            </oneWayDependantChildren>
+            <mutuallyExclusiveChildren>
+              <descendant name="mode"><value>client</value></descendant>
+              <child>local-port</child>
+              <child>local-host</child>
+              <descendant name="tls"><child>dh-params</child></descendant>
+            </mutuallyExclusiveChildren>
+            <!-- / OpenVPN client mode -->
+            <!-- OpenVPN site-to-site -->
+            <oneWayDependantChildren>
+              <dependants><descendant name="mode"><value>site-to-site</value></descendant></dependants>
+              <dependees>
+                <child>remote-host</child>
+              </dependees>
+            </oneWayDependantChildren>
+            <mutuallyExclusiveChildren>
+              <descendant name="mode"><value>site-to-site</value></descendant>
+              <descendant name="encryption"><child>ncp-ciphers</child></descendant>
+            </mutuallyExclusiveChildren>
+            <!-- / OpenVPN site-to-site -->
+            <!-- OpenVPN server mode -->
+            <mutuallyExclusiveChildren>
+              <descendant name="mode"><value>server</value></descendant>
+              <descendant name="protocol"><value>tcp-active</value></descendant>
+            </mutuallyExclusiveChildren>
+            <mutuallyExclusiveChildren>
+              <descendant name="mode"><value>server</value></descendant>
+              <child>authentication</child>
+            </mutuallyExclusiveChildren>
+            <mutuallyExclusiveChildren>
```

Comment:
This sequence of `<mutuallyExclusiveChildren>` tags is very tedious to read, to be honest.

I'm also thinking that `<mutuallyExclusiveChildren>` should only be for children that simply cannot appear together under any circumstances, like `server` and `remote-host`.

In a situation when a node _value_ is incompatible with a bunch of children, we may want to call it something else.

At the very least I'd like a more compact syntax, like:
```
<conflict>
  <source> <descendant name="mode" value="server"> </source>
  <excludes>remote-host</excludes>
  <excludes>remote-port</excludes>
  ...
</conflict>
```